### PR TITLE
CSPL-1454: Fix s1 appframework test (#582)

### PR DESCRIPTION
**Pull request by pdhanoyasplunk**
Backport for PR https://github.com/splunk/splunk-operator/pull/582.

### CODE REVIEW

File: test/s1/appframework/appframework_test.go [hunk 1]
```diff
@@ -475,6 +479,12 @@ var _ = Describe("s1appfw test", func() {
 			// Verify Monitoring Console is Ready and stays in ready state
 			testenv.VerifyMonitoringConsoleReady(deployment, deployment.GetName(), mc, testenvInstance)
 
+			// Upload apps to S3 for Standalone
+			s3TestDir := "s1appfw-" + testenv.RandomDNSName(4)
+			uploadedFiles, err = testenv.UploadFilesToS3(testS3Bucket, s3TestDir, appFileList, downloadDirV1)
```
Comment (smohan-splunk) [inline test/s1/appframework/appframework_test.go:484]: Don't we need to repopulate appFileList here?

File: test/s1/appframework/appframework_test.go [hunk 2]
```diff
@@ -697,7 +710,7 @@ var _ = Describe("s1appfw test", func() {
 			Expect(err).To(Succeed(), "Unable to deploy Standalone with App framework")
 
 			// Verify App Downlaod State on CR
-			testenv.VerifyAppListPhaseStandalone(deployment, testenvInstance, deployment.GetName(), appSourceName, enterpriseApi.PhaseDownload, appFileList)
+			// testenv.VerifyAppListPhaseStandalone(deployment, testenvInstance, deployment.GetName(), appSourceName, enterpriseApi.PhaseDownload, appFileList)
```
Comment (smohan-splunk) [inline test/s1/appframework/appframework_test.go:713]: Why did we comment this?

**Review by smohan-splunk — APPROVED** [review]

**Review by jambrosiano — APPROVED** [review]

File: test/s1/appframework/appframework_test.go [hunk 3]
```diff
@@ -911,9 +925,11 @@ var _ = Describe("s1appfw test", func() {
 			config, _ = testenv.GetAppframeworkManualUpdateConfigMap(deployment, testenvInstance.GetName())
 			Expect(strings.Contains(config.Data["Standalone"], "status: off")).To(Equal(true), "Config map update not complete")
 
+			// Verify apps downloaded by init container
```
Comment (tpereirasplunk) [inline test/s1/appframework/appframework_test.go:928]: NIT:  missing init container check

File: test/s1/appframework/appframework_test.go (hunk 1 above)
Comment (pdhanoyasplunk) [reply test/s1/appframework/appframework_test.go:484]: The app list is not changing across MC and S1 source so we are using the same variable.

File: test/s1/appframework/appframework_test.go (hunk 2 above)
Comment (pdhanoyasplunk) [reply test/s1/appframework/appframework_test.go:713]: This is verification for phase 3 will need to removed completely based on discussion with @sgontla

File: test/s1/appframework/appframework_test.go (hunk 3 above)
Comment (pdhanoyasplunk) [reply test/s1/appframework/appframework_test.go:928]: Yes identified the missed check. Needed to chat with you @tpereirasplunk before adding to understand if this was intentional or not. So we should have a check here right ?

**Review by tpereirasplunk — APPROVED** [review]

**Pull request merged**